src/Language/CSharp.ts

```diff
@@ -280,51 +287,58 @@ export class CSharpRenderer extends ConvenienceRenderer {
     }
 
     private emitClassDefinition(c: ClassType, className: Name): void {
-        this.emitType(this.descriptionForType(c), AccessModifier.Public, "partial class", className, () => {
-            if (c.properties.isEmpty()) return;
-            const blankLines = this.needAttributes && !this.dense ? "interposing" : "none";
-            let columns: Sourcelike[][] = [];
-            let isFirstProperty = true;
-            let previousDescription: string[] | undefined = undefined;
-            this.forEachClassProperty(c, blankLines, (name, jsonName, p) => {
-                const csType = this.csType(p.type, true);
-                const attribute = this.attributeForProperty(jsonName);
-                const description = this.descriptionForClassProperty(c, jsonName);
-                const property = ["public ", csType, " ", name, " { get; set; }"];
-                if (!this.needAttributes) {
-                    if (
-                        // Descriptions should be preceded by an empty line
-                        (!isFirstProperty && description !== undefined) ||
-                        // If the previous property has a description, leave an empty line
-                        previousDescription !== undefined
-                    ) {
-                        this.ensureBlankLine();
-                    }
-                    this.emitDescription(description);
-                    this.emitLine(property);
-                } else if (this.dense && attribute !== undefined) {
-                    const comment = description === undefined ? "" : ` // ${description.join("; ")}`;
-                    columns.push([attribute, " ", property, comment]);
-                } else {
-                    this.emitDescription(description);
-                    if (attribute !== undefined) {
-                        this.emitLine(attribute);
+        this.emitType(
```

Comment:
Any chance you could make this change without having to reformat the whole method? It makes it really hard to figure out what's going on in this commit if it contains changes that just reformat code.

Reply:
We're using [prettier](https://github.com/prettier/prettier) to format TypeScript.